libraries/AC_Avoidance/AC_Avoidance_config.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <AP_HAL/AP_HAL_Boards.h>
+#include <AC_Fence/AC_Fence_config.h>
+
+#ifndef AP_AVOIDANCE_ENABLED
+#define AP_AVOIDANCE_ENABLED AP_FENCE_ENABLED
```

Comment:
Not a blocker but it would be nice if simple avoidance could be enabled without a fence but with proximity sensors.

Reply:
I had a quick poke at that and it looks like we've been able to do it for a year.... just need to change `build_options.py` and the define you're pointing at here and it should be good-to-go.

I'll create a follow-up PR after this.  No idea how to test it right now...